### DISCUSSION TITLE
(It's not an issue) Using @staticmethod decorator to avoid instantiation of bound methods for each object

### DISCUSSION THREAD

**Discussion by 0xsirsaif** [encode/starlette#2544] [Potential Issue] [answered]
However, using the `@staticmethod` decorator does not have obvious preferable use cases. But I find that as long as methods that do not use `self` in their body create a bound object, that we could avoid by using `@staticmethod`

**Comment by Kludex** [comment, marked as answer]
We avoid these kinds of changes.
